pubtools/pulplib/_impl/client/client.py

```diff
@@ -239,10 +284,18 @@ def _unpack_response(cls, pulp_response):
             parsed = pulp_response.json()
         except Exception:
```

Comment:
I was looking at the Travis CI failure, which is in pylint (broad-except rule) - but you didn't introduce this code so it was unclear why it just fails now.

Then I realized the reason:  with the previous code, the exception would always be unconditionally re-raised (if a different exception wasn't raised earlier).  pylint is smart enough to figure that out.

Now the code is changed in a way which is too dynamic for pylint to consider it acceptable (from pylint's point of view, *any* kind of exception could be caught and thrown away here).  You'll need to find a way to avoid this error.

I think ideally, this method shouldn't have any logic specific to maintenance mode.

Reply:
Resolving as this now seems fixed.